### PR TITLE
WIP: Feat/success nature button

### PR DESCRIPTION
**Pull request by sericaia**
Would like some feedback about this idea as soon as possible ;)

In this PR I'm adding a new `nature: success` for the `accessible-validator-green` button. I don't have other designs yet for different selectors.

image

### CODE REVIEW

File: src/components/button/Button.module.scss [hunk 1]
```diff
@@ -106,6 +106,23 @@
         color: colors.$dash-darker-green-00;
       }
     }
+
+    &.success {
+      color: colors.$white;
+      background-color: colors.$accessible-validator-green;
+      border-color: colors.$accessible-validator-green;
+
+      // &:focus,
```
Comment (pducolin) [inline src/components/button/Button.module.scss:115]: is it expected to have no hover nor focus?

Comment (sericaia) [reply src/components/button/Button.module.scss:115]: as mentioned in the PR description I'm waiting for designs ;)

Comment (oclero) [reply src/components/button/Button.module.scss:115]: I've noticed they're often associated and merged into one visual state in web UIs, unlike desktop UIs. :)

File: src/components/button/types.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
-export type ButtonNature = 'primary' | 'secondary' | 'danger' | 'ghost';
+export type ButtonNature =
+  | 'primary'
```
Comment (oclero) [inline src/components/button/types.ts:2]: Maybe we could begin to order alphabetically, if the list begins to grow.

**Pull request closed without merging**